.tool-versions

```diff
@@ -1,4 +1,4 @@
-nodejs 16.14.0
+nodejs 16.9.1
```

Comment:
Can't we use one of the newer versions? Like 18 or something?

Reply:
I upgraded to the current version and it failed:
`Failed to install node version '18.6.0'`
It worked locally, so I'm not sure what the issue is

Reply:
Maybe some configuration on Netlify's side specifying an old version?